snap/local/etc/cruise-control/log4j.properties

```diff
@@ -0,0 +1,60 @@
+#
+# Copyright 2017 LinkedIn Corp. Licensed under the BSD 2-Clause License (the "License"). See License in the project root for license information.
+#
+
+cruisecontrol.log.level=INFO
+
+rootLogger.level=${sys:cruisecontrol.log.level}
+appenders=kafkaCruiseControlAppender, operationAppender, requestAppender
+
+property.filename=${sys:cruisecontrol.logs.dir}
```

Comment:
**niptick** maybe **logdir**? `filename` seems a bit misleading here

Reply:
I didn't choose it, went with whatever was the default in the config.